ft_volumenormalise.m

```diff
@@ -181,20 +189,13 @@
 
 % Ensure that the input MRI has interpretable units and that it is expressed in a
 % coordinate system which is in approximate agreement with the template.
+ft_notice('Doing initial alignment...')
```

Comment:
is there a reason that you now skip the processing of `cfg.initial` ? That was added only in Sept 2021 to deal with an issue reported by Arjen.

Reply:
I am not sure whether this has now been adequately fixed. It seems there is now a new cfg argument, called 'initial', which determines the functional behavior of line 196 onwards. cfg.initial does not seem to be initialized and is also not documented. So far, this option seems a bit obscure, it is not clear why (and when) it is needed.

Reply:
Looking through the GitHub issues, the `cfg.initial` issue is related to how `ft_volumenormalize` handles iEEG data. The comment from @robertoostenveld was that I had edited an older version of `ft_volumenormalize` and therefore skipped this part that was added in Sept. The block on line 192 and forward should now be doing what it is supposed to. It is just not documented what it is supposed to do. I guess it just has to be added to the documentation what `cfg.initial` is? Have I understood correctly that `cfg.initial` is supposed to be the initial alignment between iEEG electrode coordinates and the MRI volume? It seem that it can also be used if you for some reason have done a coarse initial alignment of your volume to the template MRI and therefore want to skip the initial alignment again--for some reason? If this is correct, I can just add it to the documentation?

Reply:
I updated the help on `cfg.initial`